README.md

```diff
@@ -1,92 +1,122 @@
-# OOP school library
+# Enumerable-project
 
-> In this project, I will start building my school library app. I'm practicing design patterns for building OOP applications.
-## Built With
+## Getting Started
 
-- Ruby
+# 📗 Table of Contents
 
-## Getting Started
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
```

Comment:
- [ ] I can see you don't have all the above `selected` sub-sections, so please either remove them from the `table of contents` or add the above highlighted `sub-sections` for better documentation of your project. Right now it is totally confusing and mislead any person easily. So always make sure that you have relevant information in your documentation.